_posts/2014-11-07-november-newsletter.md

```diff
@@ -302,7 +302,6 @@ Until next time. `:wq`
 
 [box]: http://en.wikipedia.org/wiki/Out_of_the_box_feature
 
-[job-rstream]: https://github.com/neovim/neovim/blob/master/src/nvim/os/rstream.c
```

Comment:
This file (and the RStream structure) doesn't exist anymore, so I just removed the link.